docs/user-guide/cli-using-benefits-of-team-config.md

```diff
@@ -0,0 +1,13 @@
+# Benefits of team configuration
+
+[Team configuration](../appendix/zowe-glossary#team-configuration) can make the initial setup of Zowe more efficient by making service connection details easier to share and maintain within dev projects.
```

Comment:
I would add "...can make the initial setup of Zowe **CLI** more efficient..."

Also, instead of "...within dev projects", it might be better to say "within your organization"